h_malloc.c

```diff
@@ -1087,6 +1087,27 @@ COLD static void handle_bugs(void) {
         ro.purge_slabs = false;
         ro.region_quarantine_protect = false;
     }
+
+    char cmdline[256];
+    if (readlink("/proc/self/cmdline", cmdline, sizeof(cmdline)) == -1) {
+        return;
+    }
+
+    static const char* broken_apps[] = {
```

Comment:
Still need to make the pointers themselves (and thus the array) constant with `static const char *const broken_apps[]`. Without that 2nd `const` this is essentially a mutable global variable due to `static`, not a constant. The code style of `char*` is also a weird Google C++ thing and we don't use that since it doesn't generalize properly and isn't good C and C++ style.